### DISCUSSION TITLE
Multilingual speaker embeddings

### DISCUSSION THREAD

**Discussion by sedol1339** [pyannote/pyannote-audio#1812] [General]
Hello! `pyannote/speaker-diarization-3.1` uses `wespeaker-voxceleb-resnet34-LM` embedding model, however according to [wespeaker docs](https://github.com/wenet-e2e/wespeaker/blob/master/docs/pretrained.md) it is only for English. Is there a multilingual embedding model for `SpeakerDiarization` constructor?

**Comment by hbredin** [comment]
It has been trained on mostly English data but will generalize quite well to other languages.

**Reply by sedol1339** [reply]
Thank you, will try it
